backend/social_connections/tests/test_telegram_commands.py

```
@@ -0,0 +1,282 @@
+
+    def test_escape(self):
+        self.assertEqual(escape('<b>&'), '&lt;b&gt;&amp;')
+
+    def test_truncate_caps_at_limit(self):
+        self.assertEqual(len(truncate('x' * 5000)), 4096)
+        self.assertEqual(truncate('short'), 'short')
+
+    def test_success(self):
+        with patch('social_connections.telegram.requests.post') as post:
+            post.return_value = self.mock_response(200, {'ok': True})
+            ok, retry_after, description = send_telegram_message('111', 'hi')
```

Ruff (0.15.21)

[warning] Unpacked variable `description` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)